packages/api/src/utils/constants.ts

```diff
@@ -0,0 +1,24 @@
+// TODO: move secret into .env
+export const jwtConstants = {
+  secret: 'secretKey',
+};
+
+export const TokenConfig = {
+  refreshToken: {
+    name: 'refresh_token',
+    keepMe: {
```

Comment:
I would rather name it with full variable name like `keepMeLogin`, after time it might be not so clear for others.